commons/com.b2international.index.test.tools/elasticsearch-default-memory-vm.options

```diff
@@ -0,0 +1,3 @@
+-Xms2147483648
+-Xmx2147483648
```

Comment:
I copied this from the `testcontainers:elasticsearch` module because it was unable to load this file from its jar during runtime. See the original file [here](https://github.com/testcontainers/testcontainers-java/blob/main/modules/elasticsearch/src/main/resources/elasticsearch-default-memory-vm.options)

Reply:
It's in bytes, didn't notice this at first glance. So 2 GB.

Reply:
Yes :)